EverestSplash/EverestSplash.cs

```diff
@@ -171,6 +186,8 @@ public class EverestSplashWindow {
         currentAssembly = GetType().Assembly;
         string serverName = EverestSplash.Name + postFix;
         Console.WriteLine("Running splash on " + serverName);
+        randomness = new Randomness();
```

Comment:
Debug type should be replaced with normal Random() and commented-out line removed.

Reply:
It is not a debug type, functionality was just implemented to enable the `forceChance` via an environment variable.